test/run-tests

```diff
@@ -978,7 +1018,8 @@ def main():
         repos = [r.split("/") for r in args.repositories.split(",")]
     setup_logging(config.get("logging", {}))
 
-    check_environment()
+    if not args.openstack:
```

Comment:
In case of citool being used, it should probably be checked in check_environment() that it is available/properly configured.

Reply:
Sure, I added a check for a config file and citool command